XenAdmin/Commands/Controls/MigrateVMToolStripMenuItem.cs

```diff
@@ -89,6 +89,10 @@ public MigrateVMCommand2(IMainWindow mainWindow, IEnumerable<SelectedItem> selec
 
             private static bool CanExecute(VM vm)
             {
+                if (vm.Connection != null && vm.Connection.IsConnected)
+                    if (Helpers.FeatureForbidden(vm.Connection, Host.RestrictInterPoolMigrate))

```

Comment:
Could we use Intra- or Cross- instead of Inter- to have the same text everywhere?